src/native_driver.jl

```diff
@@ -177,3 +206,53 @@ function Distributed.manage(manager::K8sClusterManager, id::Integer, config::Wor
         end
     end
 end
+
+function Distributed.connect(manager::K8sClusterManager, pid::Int, config::WorkerConfig)
```

Comment:
I've completely forgotten why I needed to do this but this is a stripped down version of: https://github.com/JuliaLang/julia/blob/8b19f5fa276b738034cdc26c13cc9f38ba1ea362/stdlib/Distributed/src/managers.jl#L567-L632

Reply:
Oh, things are coming back to me. When using port forwarding the standard `connect` function would make the workers connect via localhost instead of the internal network address. The modifications here allow for the some connections to use port-forwarding but the worker-to-worker connections to use the intra K8s addresses.